.github/workflows/auto-review-and-pr-summary.yml

```
@@ -0,0 +1,102 @@
+          issue-number: ${{ github.event.pull_request.number }}
+          body: "@coderabbitai review"
+
+      - name: Wait for CodeRabbit summary
+        id: wait_for_comment
+        run: |
+          for i in {1..30}; do
+            sleep 30
+            
+            # レビューコメントを取得（編集されたコメントも含む）
+            COMMENTS=$(gh api repos/$GITHUB_REPOSITORY/issues/$PR_NUMBER/comments)
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)